source/Scrapers/TeamCityQueueWaitScraper.cs

```diff
@@ -158,6 +158,8 @@ static Build[] GetFilteredQueuedBuilds(TeamCityClient teamCityClient)
                 // exclude builds just waiting on other builds
                 .Where(qb => !qb.WaitReason.Contains("Build dependencies have not been built yet"))
                 .Where(qb => !branchWaitRegex.IsMatch(qb.WaitReason))
```

Comment:
Consider adding a comment for this (to be in sync with the other ones)

Reply:
Both are regarding concurrency, so I moved the comment and specified that it meant the 2 checks